packages/origin-247-certificate/src/offchain-certificate/config/configuration.ts

```diff
@@ -0,0 +1,20 @@
+import { Configuration } from './config.interface';
+
+const configuration = (): Configuration => ({
```

Comment:
I am not really convinced. You migrated all variables here, yet you use only two of them.

Reply:
I've migrated all the places to use that configuration instead of directly accessing `process.env`